Factory Pattern/src/OnlineOrderProcessingFactory.java

```diff
@@ -0,0 +1,13 @@
+public class OnlineOrderProcessingFactory {
+    public static Order createOrder(String orderType) {
```

Comment:
what is APi consumer sends `food` or `FOOD` instead of `Food`. If u use `enum` u can restrict the orderType